app/src/main/java/app/revanced/manager/patcher/PatcherProgress.kt

```diff
@@ -0,0 +1,80 @@
+package app.revanced.manager.patcher
+
+import android.os.Parcelable
+import kotlinx.parcelize.Parcelize
+
+
+@Parcelize
+sealed class ProgressEvent : Parcelable {
+    abstract val stepId: StepId?
+
+    @Parcelize
+    data class Started(override val stepId: StepId) : ProgressEvent()
+
+    @Parcelize
+    data class Progress(
+        override val stepId: StepId,
+        val done: Long? = null,
```

Comment:
`current` would be a better name in my opinion. `done` sounds like the name for a boolean.